.github/workflows/kuttl-int-tests.yaml

```diff
@@ -15,8 +14,7 @@ jobs:
 
     - uses: actions/setup-go@v5
       with:
-        go-version-file: go.mod
-        check-latest: true
+        go-version: 1.22.0
```

Comment:
> Don't know if there is a better way to fix it

checkout of the repo needs to come first because `setup-go` looks for `go.mod` in the checked out code.